src/Helpers/Composer/PackageHelper.php

```diff
@@ -70,5 +71,41 @@ private function getComposer(InputInterface $input, OutputInterface $output) {
         return $composer;
     }
 
+    /**
+     * @param PackageHelper $packageHelper
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return bool
+     */
+    public static function isPhpUnitInstalled(
```

Comment:
I don't like any of these changes. 
  * There is no benefit to these classes being static. Why would I want to make consumers of this class pass instances of itself into itself? If it needs itself it can use $this. 
  * isPhpUnitInstalled, isbehatInstalled, isTravisInstalled, isSlimInstalled, isZf2Installed, isSymfonyInstalled, when does this class stop growing following the pattern that we add shortcut methods onto this class?
  * The shortcut methods aren't worth the gain. It saves one parameter argument (that is optional) at the expense of functional overhead and a extensional pattern that doesn't scale (see first bullet)
  
A more extendable pattern might be package helper decorator per package you want to provide a shortcut for. Also, that decorator likely belongs somewhere under the namespace of the command as its not safe to say all things will need "any version" of "package" (what you ask for with the null parameter).